README.rst

```diff
@@ -20,33 +20,85 @@ configurations remain synchronized with the state of all databases in the
 cluster. PgQ is used for buffering mutation records until they are collected by
 the relay and forwarded to their final destination.
 
+Usage
+=====
+
+PGShovel is a tool which replicates mutations (``INSERT``, ``UPDATE`` and ``DELETE`` operations) made to a PostgreSQL data out of the database and into an ordered stream of transaction-aware mutation messages.  These messages can then be consumed by 3rd party systems as needed.  Commonly, the mutation stream is consumed by data warehouse operations, cache updating services, search index services or can even be processed into other new streams.
+
+At Disqus PGShovel is actively being developed to replicate changes from our master Postgres database - which contains the core commenting and other relational data - into our data warehousing system. There is further planned work to also consume the mutation stream which is models mutations to rows and columns into a new stream of **object snapshots**.  Unlike mutations, these snapshots accurately and succinctly model our business objects and are free from the idiosyncrasies of attempting to read and interpret their serialized structure in the rows and columns of an RDBMS.
```

Comment:
Yeah I actually waffled on not including this sentence since it would take like 4-6 sentences to explain why it's a good idea, and I felt like that was a bad thing to do so early in the README.  I will probably just make this sentence even simpler and disarming.